services/ui-src/src/components/cards/EntityCardBottomSection.tsx

```diff
@@ -1,12 +1,10 @@
-// components
-import { Text } from "@chakra-ui/react";
 // utils
 import { AnyObject } from "types";
 
 export const EntityCardBottomSection = ({ entityType }: Props) => {
   switch (entityType) {
     default:
-      return <Text>{entityType}</Text>;
+      return <></>;
```

Comment:
what is this file doing if it's default is to return a fragment?

Reply:
This will be for PDF work that we haven't started or brought over from MCR. It's simply living there for now so we can add to it in the future.